src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,23 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
-);
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Todos = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Todos> = (props) => {
+  const { todos } = props;
+
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <div
+          key={todo.id}
+        >
+          <TodoInfo todo={todo} />
+        </div>
```

Comment:
_comment_
We were not told why do we need that unique `key` for every element. But I assume it was meant to be a property of called Component not a wrapper
```suggestion
        <TodoInfo todo={todo} key={todo.id} />
```

Reply:
Yes, I agree with you, it's unnecessary to wrap components. I didn't get to use the key inside component attribute)